### ISSUE TITLE
Illegal characters in path.

### ISSUE THREAD

**Issue by Aurion72** [gaetansenn/xhr-cache#23]
Hi,

Im trying to use this module in my projet but i got an issue with fetching data
 
![image](https://user-images.githubusercontent.com/17456821/107193921-b0d85580-69ef-11eb-83dd-8ed31a3dcf66.png)

Someone can told me what is wrong ? :)

**Comment by gaetansenn** [comment]
Hello @Aurion72,

Look's like you are using the module on windows operating system. This is probably the reason of the error.
Unfortunately I don't have a windows instance to fix your issue. Can you upload an exemple of your xhr-cache configuration ?
Regards,

Gaetan

**Comment by Aurion72** [comment]
Yes it is, im using windows 10.

This is my configuration :

```
 xhrCache: {
    resources: [{
      name: 'categories',
      maxAge: 3600 * 1000, // TTL of resource
      init: true, // Fetch the resource at nuxt start
      request: {
        method: 'get',
        url: process.env.API_ENDPOINT + '/categories'
      }
    }]
  },
```

**Comment by Aurion72** [comment]
@gaetansenn It seems to be the "\" from this path that causes the problem.

![image](https://user-images.githubusercontent.com/17456821/107483880-f4fb5f80-6b81-11eb-846b-862a05d5ccae.png)

It would be interesting to solve this problem because I don't think I'm the only one to use windows :D and this module sounds very interesting and I think I can do a lot of things with it :(

**Comment by gaetansenn** [comment]
I don't have time to fix this error that will only occur in dev mode.
What we can do as I don't have a window computer is that you can try to debug the library. The simply way is to console.log directly from the node_modules package and localize the problem. I think this is a problem of compatibility of a node module that is not working for window operating system.

**Comment by gaetansenn** [comment]
Available in release v3.1.1
Tks to @mlauret for his contribution
